pyxlma/lmalib/flash/properties.py

```diff
@@ -225,14 +226,10 @@ def flash_stats(ds):
                                                             df['event_y'],
                                                             df['event_z']))
 
-    #Compute flash discharge energy using parallel plate capacitor: (probably
-    #a better way to get the areas without having to have an embedded function
-    #call?)
+    #Compute flash discharge energy using parallel plate capacitor
     event_energy = fl_gb.apply(lambda df: event_discharge_energy(df['event_z'],
-                                                                 event_hull_area(df['event_x'],
-                                                                                 df['event_y'],
-                                                                                 df['event_z'])
-                                                                ))
+                                                                 event_area[df['event_parent_flash_id']]))
```

Comment:
Looks like you didn't need to do the indexing of the first point (as discussed offline) since the later assignment to `ds` takes care of the first point section?

Reply:
I still do, it's in the event_discharge_energy function now on line 126 where I use area.iloc[0].